cli/azd/cmd/testdata/TestUsage-azd-infra.snap

```diff
@@ -5,7 +5,9 @@ Usage
   azd infra [command]
 
 Available Commands
-  synth	: Write IaC for your project to disk, allowing you to manage it by hand. (Alpha)
+  gen     	: Write IaC for your project to disk, allowing you to manage it by hand.
+  generate	: Write IaC for your project to disk, allowing you to manage it by hand.
+  synth   	: Write IaC for your project to disk, allowing you to manage it by hand.
```

Comment:
Can we only show generate here, and make the others hidden?
Similar to how `azd extension` is the one shown in help, but `azd ext` also works.

Also, can you please rebase on top of https://github.com/JeffreyCA/azure-dev/tree/jeffreyca/5231/compose-beta instead of main? There are also other references to infraSynth and infra synth that need to be updated, like the next-steps.mdt files